assets/src/dashboard/components/card-grid-item/index.js

```diff
@@ -41,11 +42,14 @@ const StyledGridItem = styled.div`
   }
 `;
 
-const CardGridItem = ({ children, ...rest }) => {
-  return <StyledGridItem {...rest}>{children}</StyledGridItem>;
-};
+const CardGridItem = ({ children, ...rest }) => (
+  <StyledGridItem {...rest}>{children}</StyledGridItem>
+);
```

Comment:
This looks like just a passthrough right now. Could we export StyledGridItem as CardGridItem?